.github/actions/smoke-test/build.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+TEMPLATE_ID="$1"
+
+set -e
+
+shopt -s dotglob
+
+SRC_DIR="/tmp/${TEMPLATE_ID}"
+cp -R "src/${TEMPLATE_ID}" "${SRC_DIR}"
+
+pushd "${SRC_DIR}"
+
+# Configure templates only if `devcontainer-template.json` contains the `options` property.
+OPTION_PROPERTY=( $(jq -r '.options' devcontainer-template.json) )
+
+if [ "${OPTION_PROPERTY}" != "" ] && [ "${OPTION_PROPERTY}" != "null" ] ; then  
```

Comment:
Prefer [[ ]] over []

https://google.github.io/styleguide/shellguide.html#test----and---